### DISCUSSION TITLE
Two ext4 partitions : one for root ( '/' ) and another for home ( '/home' )

### DISCUSSION THREAD

**Discussion by RayZ0rr** [picodotdev/alis#203] [Q&A] [answered]
We can use `PARTITION_CUSTOM_PARTED_UEFI` to make separate partitions. But how to mount ext4 partition for home, say `/dev/sda3` on `/home` or `/mnt/home`?

**Comment by picodotdev** [comment, marked as answer]
The existing options to customize the partition scheme was quite limited that did not support home on separate partition.

I will add fully configurable custom partition scheme and mountpoints feature. Already implemented, need to test it.

https://github.com/picodotdev/alis/pull/204

Merged to master

**Reply by picodotdev** [reply]
I already did basic testing, if you want to give a try. At the moment in the sid branch. All partitions must to be in the same device, is that enough for you?

With home on its own partition.

![1](https://user-images.githubusercontent.com/720376/159137103-894c8492-2e06-4306-84fd-3e28473dad0e.png)

**Reply by RayZ0rr** [reply]
Sorry for the late reply. But this is good feature for now. However, would providing something like `EXT4_MOUNTPOINTS` variable in [alis.conf](https://github.com/picodotdev/alis/blob/master/alis.conf) be useful? This is similar to `BTRFS_SUBVOLUMES_MOUNTPOINTS` but for ext4 which makes it easy to mount partitions from separate disks.

**Reply by RayZ0rr** [reply]
Now, I see that there's `DEVICE` variable is there to select single device. Then I think `PARTITION_MOUNT_POINTS` is enough. No need for others.
